src/grandorgue/sound/scheduler/GOSoundScheduler.cpp

```diff
@@ -34,15 +34,9 @@ void GOSoundScheduler::Clear() {
   Unlock();
 }
 
-void GOSoundScheduler::Lock() {
-  m_ItemCount = 0;
-  __sync_synchronize();
-}
+void GOSoundScheduler::Lock() { m_ItemCount = 0; }
```

Comment:
Could you move this short function to the header file

Reply:
Done